modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/XMLStreamReaderUtil.java

```diff
@@ -55,6 +57,8 @@ public static boolean moveToEndOfElement(XMLStreamReader xtr, String elementName
                     if (xtr.getLocalName().equals(elementName))
                         return true;
                     break;
+                default:
```

Comment:
These 2 lines of code are already at master; you need to rebase your master branch before adding new code.